src/index.tsx

```diff
@@ -191,7 +219,7 @@ export default class SwipeCards extends Component<Props, State> {
     this._panResponder = PanResponder.create({
       onMoveShouldSetPanResponderCapture: (e, gestureState) => {
         if (Math.abs(gestureState.dx) > 3 || Math.abs(gestureState.dy) > 3) {
-          this.props.onDragStart()
+          this.props.onDragStart!()
```

Comment:
the little bang `!` after the method name here is a TS operator that says, "I know better than you do, compiler, and I guarantee that it will never be undefined." As you can imagine, it's a dangerous operator, but I can't figure out another way around the dilemma right now.

Reply:
Typescript operators in the between a function name and the invoking parentheses – yuck!